locales/en/landing.ftl

```diff
@@ -52,3 +52,5 @@ depends-on-hackers = Not all breaches expose all the same info. It just depends
 ## See if you’ve been in a data breach
 
 see-if-data-breach = See if you’ve been in a data breach
+hibp-footer-attribution = Breach data provided by <b>{ -brand-HIBP }</b>
```

Comment:
We have already 3 strings for this text:
* `hibp-attribution` in `app.ftl`
* `exposure-landing-result-footer-attribution` in `exposure-scan.ftl`
* `email-2022-hibp-attribution` in `email-strings.ftl`

Funny enough, each one of them uses a different format for that link (and this one would add a fourth).

Are these inconsistencies necessary, because of the code, or can we figure out a way to have more consistency and less duplication (e.g. reusable modules like a footer)?

Reply:
Interesting&hellip; great catch!

```sh
git grep -n "Breach data provided by " locales/en/ | grep -v ":#"

locales/en/app.ftl:87:hibp-attribution = Breach data provided by { $hibp-link }
locales/en/email-strings.ftl:98:email-2022-hibp-attribution = Breach data provided by <a { $hibp-link-attr }>{ -brand-HIBP }</a>
locales/en/exposure-scan.ftl:23:exposure-landing-result-footer-attribution = Breach data provided by <hibp-link>{ -brand-HIBP }</hibp-link>
locales/en/landing.ftl:55:hibp-footer-attribution = Breach data provided by <b>{ -brand-HIBP }</b>
```

Although, not sure if some of these are orphaned references due to the redesign.

<details>
<summary><code>hibp-attribution</code> _might_ be unused</summary>

> ```ini
> locales/en/app.ftl:87:hibp-attribution = Breach data provided by { $hibp-link }
> ```
> 
> ```sh
> git grep "hibp-attribution" -- src ':!*.css' | cat
> 
> # src/views/emails/email2022.js:        ${getMessage('email-2022-hibp-attribution', {
> # src/views/partials/breachDetail.js:      ${getMessage('email-2022-hibp-attribution', {
> # src/views/partials/exposureScan.js:      <p class="hibp-attribution">
> # src/views/partials/exposureScan.js:      <p class="hibp-attribution">
> # src/views/partials/exposureScan.js:      <p class="hibp-attribution">
> # src/views/partials/landing.js:  <p class='hibp-attribution'>${getMessage('hibp-footer-attribution')}</p>
> ```
</details>

<details>
<summary><code>email-2022-hibp-attribution</code> has 2 usages</summary>

> ```ini
> locales/en/email-strings.ftl:98:email-2022-hibp-attribution = Breach data provided by <a { $hibp-link-attr }>{ - brand-HIBP }</a>
> ```
> 
> ```sh
> git grep "email-2022-hibp-attribution" -- src ':!*.css' | cat
> 
> # src/views/emails/email2022.js:        ${getMessage('email-2022-hibp-attribution', {
> # src/views/partials/breachDetail.js:      ${getMessage('email-2022-hibp-attribution', {
> ```
</details>


<details>
<summary><code>exposure-landing-result-footer-attribution</code> has 3 usages</summary>

> ```ini
> locales/en/exposure-scan.ftl:23:exposure-landing-result-footer-attribution = Breach data provided by <hibp-link>{ -brand-HIBP }</hibp-link>
> ```
> 
> ```sh
> git grep "exposure-landing-result-footer-attribution" -- src ':!*.css' | cat
> 
> # src/views/partials/exposureScan.js:        ${getMessage('exposure-landing-result-footer-attribution').replace('<hibp-link>', '<a href="https://haveibeenpwned.com/" target="_blank">').replace('</hibp-link>', '</a>')}
> # src/views/partials/exposureScan.js:        ${getMessage('exposure-landing-result-footer-attribution').replace('<hibp-link>', '<a href="https://haveibeenpwned.com/" target="_blank">').replace('</hibp-link>', '</a>')}
> # src/views/partials/exposureScan.js:        ${getMessage('exposure-landing-result-footer-attribution').replace('<hibp-link>', '<a href="https://haveibeenpwned.com/" target="_blank">').replace('</hibp-link>', '</a>')}
> ```
</details>

<details>
<summary><code>hibp-footer-attribution</code> has 1 usage</summary>

> ```ini
> locales/en/landing.ftl:55:hibp-footer-attribution = Breach data provided by <b>{ -brand-HIBP }</b>
> ```
> 
> ```sh
> git grep "hibp-footer-attribution" -- src ':!*.css' | cat
>
> # src/views/partials/landing.js:  <p class='hibp-attribution'>${getMessage('hibp-footer-attribution')}</p>
> ```
</details>